ads/google/a4a/traffic-experiments.js

```diff
@@ -165,9 +173,24 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '0': null, // TODO Ensure does not generate exp id
     '1': controlBranchId,
     '2': treatmentBranchId,
+    '3': hasLaunched(win, element) ? treatmentBranchId : controlBranchId, // DF
```

Comment:
Why do we need special hasLaunched behavior here?  I would assume this gets a new experiment ID (e.g. controlWithDelayedId)